demos/appyx-navigation/src/main/kotlin/com/bumble/appyx/navigation/node/spotlight/SpotlightNode.kt

```diff
@@ -79,6 +85,17 @@ class SpotlightNode(
                         color = Color.Black,
                         fontWeight = FontWeight.Bold
                     )
+
+                    val value = getMotionPropertyRenderValue<DpOffset, Position>()
+                    if (value != null) {
+                        Text(
+                            modifier = Modifier.align(Alignment.Center),
+                            text = value.x.toString(),
+                            fontSize = 21.sp,
+                            color = Color.Black,
+                            fontWeight = FontWeight.Bold
+                        )
+                    }
```

Comment:
Same, I wouldn't add it to the default Spotlight demo.

Reply:
fixed